src/webui/www/private/views/filters.html

```diff
@@ -263,7 +263,7 @@
         });
 
         document.getElementById("Filters_pad").addEventListener("dblclick", (event) => {
-            if (localPreferences.get("dblclick_filter", "1") !== "1")
+            if (window.qBittorrent.ClientData.get("dblclick_filter") !== "1")
```

Comment:
```suggestion
            if ((window.qBittorrent.ClientData.get("dblclick_filter") ?? "1") !== "1")
```

Reply:
The nullish coalescing is unnecessary - it doesn't change anything given the check against `!== "1"`